src/main/java/com/spotinst/sdkjava/model/ApiEbsDevice.java

```diff
@@ -17,14 +18,16 @@
 @JsonFilter("PartialUpdateEntityFilter")
 class ApiEbsDevice implements IPartialUpdateEntity {
```

Comment:
Should move this file to BL. Is this class used somewhere else also?

Reply:
This class was shared with ElastigroupConverter.  I will create a new one for VNG and move to the API folder